components/Banner/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -235,6 +239,25 @@ - (void)testLongTextWithTwoActionsLTR {
   [self generateSnapshotAndVerifyForView:self.bannerView];
 }
 
+- (void)testLongTextWithTwoActionsWithTwoOrientationChangeActions {
+  // When
```

Comment:
If this is an iOS 13 snapshot, why does it not guard against execution on early XCode/iOS versions?  From the snapshot code, it looks like specifying "YES" on an earlier device will result in the test silently not executing.

```
- (void)snapshotVerifyView:(UIView *)view
                 tolerance:(CGFloat)tolerancePercent
              supportIOS13:(BOOL)supportIOS13 {
  if (!supportIOS13 && ![self isSupportedDevice]) {
    return;
  } else if (supportIOS13 && ![self isSupportedIOS13Device]) {
    return;
  }
```

Would you please add a comment to either the test method or the "Then" section to indicate why this test should not run on earlier iOS versions?

Reply:
Good points! Will do.